client/blocks/login/login-form.jsx

```diff
@@ -859,9 +861,14 @@ export class LoginForm extends Component {
 					<p className="login__form-terms">{ socialToS }</p>
 					{ isWoo && ! isPartnerSignup && this.renderLostPasswordLink() }
 					<div className="login__form-action">
-						<FormsButton primary disabled={ isSubmitButtonDisabled }>
+						<Button
+							type="submit"
+							primary
+							busy={ isSubmitButtonDisabled }
```

Comment:
isSubmitButtonDisabled also considers if the email is bein sent

Reply:
I don't it's a good idea to use this prop for busy. When you first land on the page now, the button is busy.